### PR TITLE
Create entities

### PR DESCRIPTION
**Pull request by thamoshi**
Created the Entities for the project

### CODE REVIEW

File: src/main/kotlin/br/com/creditas/projethom/model/Employee.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package br.com.creditas.projethom.model
+
+import java.time.LocalDateTime
+import java.util.*
```
Comment (pmagalhaes2) [inline src/main/kotlin/br/com/creditas/projethom/model/Employee.kt]: avoid wildcard

File: src/main/kotlin/br/com/creditas/projethom/model/System.kt [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package br.com.creditas.projethom.model
+
+import java.time.LocalDateTime
+import java.util.*
```
Comment (pmagalhaes2) [inline src/main/kotlin/br/com/creditas/projethom/model/System.kt]: avoid wildcard

File: src/main/kotlin/br/com/creditas/projethom/model/Employee.kt (hunk 1 above)
Comment (GuilhermeeSette) [inline src/main/kotlin/br/com/creditas/projethom/model/Employee.kt]: Não é uma boa prática usar importações com estrela. Costumamos importar item a item.

**Review by GuilhermeeSette — APPROVED** [review]

**Comment by GuilhermeeSette** [comment]
🚀

**Review by pmagalhaes2 — APPROVED** [review]

File: src/main/kotlin/br/com/creditas/projethom/model/System.kt [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+data class System(
+  @Id
+  val id: UUID = UUID.randomUUID(),
+  val name: String,
+  @ManyToOne
+  val owner: Team? = null,
+  val documentation: String,
+  val url: String,
+  val createdAt: LocalDateTime = LocalDateTime.now(),
+  var updatedAt: LocalDateTime = LocalDateTime.now()
```
Comment (andreh-silva) [inline src/main/kotlin/br/com/creditas/projethom/model/System.kt:19]: its works, but is usual to use a column annotation to describe the name. But its ok

File: src/main/kotlin/br/com/creditas/projethom/model/Team.kt [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  val name: String,
+  val description: String,
+  val tribe: Tribe,
+  @OneToMany
+  val members: List<Employee> = ArrayList(),
+  @OneToMany
+  val systems: List<System> = ArrayList(),
+  val createdAt: LocalDateTime = LocalDateTime.now(),
+  var updatedAt: LocalDateTime = LocalDateTime.now()
+)
```
Comment (andreh-silva) [inline src/main/kotlin/br/com/creditas/projethom/model/Team.kt]: EOF in a last empty line

Comment (andreh-silva) [reply src/main/kotlin/br/com/creditas/projethom/model/Team.kt]: tip: to validate and fix this type of issue you can add detekt in your project.

**Pull request merged**